velox/exec/tests/TableWriteTest.cpp

```diff
@@ -2947,6 +2947,53 @@ DEBUG_ONLY_TEST_P(
       "Aborted for external error");
 }
 
+DEBUG_ONLY_TEST_P(UnpartitionedTableWriterTest, dataSinkAbortError) {
+  if (fileFormat_ != FileFormat::DWRF) {
```

Comment:
Why is this tests not running for DWRF?

Reply:
The test injects error in dwrf writer so it only applies for that. Will add a comment.